src/programs/BuddyProject.ts

```diff
@@ -7,21 +7,27 @@ import {
 import { Repository } from "typeorm";
 
 const updateDatabaseWithQuery = (
-  BE: Repository<BuddyProjectEntry>,
+  BuddyEntryRepo: Repository<BuddyProjectEntry>,
   memberId: string,
   buddyId: string,
   BuddyEntry: BuddyProjectEntry
 ) => {
-  BE.createQueryBuilder()
+  BuddyEntryRepo.createQueryBuilder()
     .update(BuddyEntry)
     .set({ matched: true, buddy_id: buddyId })
     .where("used_id = :member_id", { member_id: memberId })
```

Comment:
That is a big problem.

```suggestion
    .where("user_id = :member_id", { member_id: memberId })
```